backend/tests/Feature/OnboardingWizardTest.php

```
@@ -110,6 +110,54 @@ public function test_step3_creates_template_with_steps(): void
         $this->assertEquals(3, ProcessTemplate::first()->steps()->count());
     }
 
+    public function test_step3_resubmit_does_not_create_duplicate_template(): void
+    {
+        $pt = ProductType::factory()->create();
```

PHPStan (2.2.6)

[error] Call to an undefined static method App\Models\ProductType::factory().


(staticMethod.notFound)